<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuDestinos</class>
 <widget class="QWidget" name="MenuDestinos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>401</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Agregar</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>381</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="inputCodigoD">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Codigo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputNombreD">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Nombre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputTipoD">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Tipo &quot;E&quot; entrada o &quot;N&quot;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputEstatusD">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Estatus &quot;A&quot; o &quot;I&quot;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonAceptarAgregar">
            <property name="text">
             <string>Aceptar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modificar</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Mostrar">
    <attribute name="title">
     <string>Mostrar</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Eliminar</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
